cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -191,37 +197,89 @@ def enable_disable_migration_ui(appliance):
 @pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs'],
                             ['nfs', 'iscsi'], ['iscsi', 'iscsi']], indirect=True)
 def test_single_datastore_single_vm_mapping_crud(appliance, enable_disable_migration_ui,
-                                            create_provider, form_data_single_datastore):
-    # TODO: Add "Delete" method call.This test case does not support update/delete
-    # as update is not a supported feature for mapping,
-    # and delete is not supported in our automation framework.
+                                            providers, form_data_single_datastore,
```

Comment:
**REQUIRED:** So, I see a few things here.
1) There are 3 tests here which are identical. We can write one function that will do all of them, allowing you to only have changed this in one single place.
2) I was looking at if we could make them a single test, but I actually like them being separate.

Reply:
@psav I am soon going to change them to actual migration test, they will create and migrate VMs and not only remain crud, I do no need so many CRUD tests anyway. so previously @mshriver had suggested me to mark these crud for now. so I will request if we can keep those as is now, and will update those in my next PR.